lib/common.js

```diff
@@ -294,7 +294,11 @@ exports.createTableForEnvironments = function (environments) {
       targets.push(target._id);
     });
 
-    table.push([env._id, env.label, env.autoDeployOnCreate, env.autoDeployOnUpdate, targets.join(", "), moment(env.modified).fromNow()]);
+    // Display undefined options as 'false'
+    var autoDeployOnCreate = env.autoDeployOnCreate || false;
+    var autoDeployOnUpdate = env.autoDeployOnUpdate || false;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the literal "false" boolean value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS1125)